mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
```

Comment:
Note the hint here, it says you already created a function that calculates the sales tax. 
So in the function `addTaxAndFormatCurrency` we want you to use the function `calculateSalesTax` to calculate what the new price should be.

Again, look at the name of the function, it gives you a clue to what the logic should be.